src/main/java/domain/AttendanceTimes.java

```diff
@@ -0,0 +1,103 @@
+package domain;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class AttendanceTimes {
+
+    private List<AttendanceTime> attendanceTimes;
```

Comment:
```suggestion
    private final List<AttendanceTime> attendanceTimes;
```
변경되지 않는다면 final로 선언해도 좋을 것 같아요.